go/test/endtoend/onlineddl/vrepl/onlineddl_vrepl_test.go

```diff
@@ -45,6 +45,9 @@ var (
 	httpClient       = throttlebase.SetupHTTPClient(time.Second)
 	throttlerAppName = "vreplication"
 
+	normalMigrationWait   = 20 * time.Second
+	extendedMigrationWait = 20 * time.Second
```

Comment:
The increased timeouts are not strictly relevant to fixing the race condition, but rahter just to allow for some GitHub CI slowness.